pages/common/stripe.md

```diff
@@ -0,0 +1,24 @@
+# stripe
+
+> Interact with your Stripe account.
+> More information: <https://github.com/stripe/stripe-cli>.
+
+- Log all activity on your account:
```

Comment:
I believe this tails the logs right ? In that case "Log all activity" might be incorrect because it does not log, rather views the logs. Also, we should remove "your", as we avoid 2nd person in our pages.

Reply:
Wow, I must have missed these